src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,58 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public int splitAndSum(String value)throws Exception{
+        int result = checkNull_Empty(value);
+        String[] values;
+
+        if(result == 0){
+            return result;
+        }else{
+            values = checkSeparator(value);
+            int calcul = 0;
+            for(String val : values){
+                calcul += checkNegative(Integer.valueOf(val));
+            }
+            result = calcul;
+        }
+        return result;
+    }
+
+    public int checkNull_Empty(String value)throws Exception{
+        int result = -1;
+
+        if(value == null){
+            result = 0;
+        }else if(value.isEmpty()){
+            result = 0;
+        }
+        return result;
+    }
+
+    public String[] checkSeparator(String value) throws Exception{
+
+        String[] number;
+
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(value);
+
+        if(m.find()){
```

Comment:
공백 라인을 적절한 위치에 사용함에 따라 문맥을 구분 시켜 가독성을 높일수 있어요. 
```suggestion
        String[] number;
        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(value);

        if(m.find()){
```